Directory.Build.props

```diff
@@ -19,7 +19,8 @@
   <!-- Define vNext UAP Moniker -->
   <PropertyGroup>
     <UAPvNextVersion>10.0.16300</UAPvNextVersion>
-    <UAPvNextTFMFull>UAP,Version=v$(UAPvNextVersion)</UAPvNextTFMFull>
+    <UAPvNextTFI>UAP</UAPvNextTFI>
```

Comment:
out of curiosity: what's TFI? Target Framework Identifier?

Reply:
right.